src/server/AppManager.ts

```diff
@@ -878,6 +893,7 @@ export class AppManager {
         this.accessorManager.purifyApp(app.getID());
         this.uiActionButtonManager.clearAppActionButtons(app.getID());
         this.videoConfProviderManager.unregisterProviders(app.getID());
+        this.oauthAppsManager.removeAllOAuthApps(app.getID());
```

Comment:
I'm not so sure about this change. OAuthApps are entities in Rocket.Chat, like rooms, and we don't remove all rooms created by apps when we uninstall it. I'm inclined to say we shouldn't remove it. Wdyt?

Reply:
I am very open on this, actually, I am not sure on this and I am ok on removing this part